lib/src/components/_internal_components.dart

```diff
@@ -295,6 +295,9 @@ class PressDetector extends StatelessWidget {
   /// Called when user long press on calendar.
   final DatePressCallback? onDateLongPress;
 
+  /// Called when user taps on calendar.
```

Comment:
Documentation for this variable is not clear. From the variable name it suggests that if user taps on any of the date cell then it will give a callback so add better wording for the comment.